modules/RTC/JitsiLocalTrack.js

```diff
@@ -332,22 +332,6 @@ JitsiLocalTrack.prototype.isMuted = function () {
 };
 
 /**
- * Private method. Updates rtc property of the track.
- * @param rtc the rtc instance.
- */
-JitsiLocalTrack.prototype._setRTC = function (rtc) {
-    this.rtc = rtc;
-    // We want to keep up with postponed events which should have been fired
-    // on "attach" call, but for local track we not always have the conference
-    // before attaching. However this may result in duplicated events if they
-    // have been triggered on "attach" already.
-    for(var i = 0; i < this.containers.length; i++)
-    {
-        this._maybeFireTrackAttached(this.containers[i]);
-    }
```

Comment:
_setRTC was called in addLocalTrack, and now in addLocalTrack we don't call _setConference(which now calls _maybeFireTrackAttached), which means we do not call _maybeFireTrackAttached at that moment. Is this ok?